src/gui_common/GUICommon.cs

```diff
@@ -80,6 +80,24 @@ public void TweenBarValue(TextureProgress bar, float targetValue, float maxValue
         tween.Start();
     }
 
+    public void ModulateFadeIn(Control control, float duration)
```

Comment:
Is there a chance that this gets called on another tooltip while something is still fading? Does this break in that case?
That's basically my only comment on this (I didn't playtest this yet).

Reply:
> Is there a chance that this gets called on another tooltip while something is still fading? 

Probably, might not happen very often but it should be fine nonetheless.

> Does this break in that case?

To my knowledge, no. I tested calling the method for two different controls at the same time and they both fade in nicely, so it seems tween nodes could execute multiple interpolate calls with no problem.

Reply:
That's good. I'll do a quick playtest later today, and merge if I don't see any problems.